docs/_integration-tutorial-code/example-eslint-integration.js

```diff
@@ -0,0 +1,64 @@
+/** 
+ * @fileoverview An example of how to integrate ESLint into your own tool
+ * @author Ben Perlmutter
+ */
+
+const { ESLint } = require("eslint");
+
+function createESLintInstance(eslintConfig){
+    if (eslintConfig) {
+        return new ESLint({ useEslintrc: false, overrideConfig: eslintConfig });
+    } else {
+        return new ESLint({ useEslintrc: false });
+    }
+}
+
+// Define a function that lints the specified files and returns the error results
+async function getLintingResults(eslint, filePaths) {
```

Comment:
`getLintingResults` doesn't seem like the right name here because we are doing more than just retrieve a value. (Running ESLint and also writing fixes to disk.)

Maybe just `lintAndFix()`?

Reply:
changing